README.md

```diff
@@ -1,4 +1,4 @@
-# jml
+# jml ![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/jabacat/jml/build-test.yml?style=for-the-badge)
 JABACAT-created machine learning library from scratch.
```

Comment:
Small nitpick: I think it would look better if it was on its own line.
```suggestion
# jml
JABACAT-created machine learning library from scratch.\
\
![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/jabacat/jml/build-test.yml?style=for-the-badge)
```